service/src/main/kotlin/no/nav/su/se/bakover/service/søknadsbehandling/SøknadsbehandlingServiceImpl.kt

```diff
@@ -352,6 +355,9 @@ internal class SøknadsbehandlingServiceImpl(
                     val vedtak = Vedtak.fromSøknadsbehandling(iverksattBehandling, utbetaling!!.id, clock)
                     vedtakRepo.lagre(vedtak)
 
+                    val kontrollsamtale = Kontrollsamtale.fromVedtak(vedtak)
```

Comment:
Er det for tidlig å spørre om hva som skjer ved opphør? Da vil en kunne lage ny stønadsperiode som overlapper med den forrige. Slik at kontrollsamtalene "overlapper".

Reply:
Ved opphører skal vi endre den planlagte kontrollsamtalen sin status til `annuler` - mao, ingen planlagte kontrollsamtaler er lagt opp. Dersom brukeren da søker igjen, så vil en ny kontrollsamtale opprettes